content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -30,9 +30,41 @@ To enable debug mode for the Datadog Java Tracer, set the flag `-Ddd.trace.debug
 
 {{< programming-lang lang="python" >}}
 
-To enable debug mode for the Datadog Python Tracer, set the environment variables `DD_TRACE_DEBUG=true` and `DD_CALL_BASIC_CONFIG=true` when using `ddtrace-run`.
+To enable debug mode for the Datadog Python Tracer, review the instructions below for the specific scenario that applies to the application being traced. Logs emitted by the Python Tracer contain the name **[ddtrace]**.
+
+#### 1.3.0 or higher
+
+1. Set these environment variables to enable debug mode:
+    - `DD_TRACE_DEBUG=true`
+    - `DD_TRACE_LOGGING_RATE=0` (Optional: used for more tracer logs)
```

Comment:
```suggestion
```

Not needed, this is now implied when setting `DD_TRACE_DEBUG=true`

Reply:
Since 0.43.0

Reply:
Marking this as resolved !